Ports/rvvm/package.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='rvvm'
+version='0.5'
+archive_hash='3a1dbb91ad04f068078bc6c6c27cc5792eebc111907cb5a14bde158fe6e757c9'
+files="https://github.com/LekKit/RVVM/archive/v${version}.tar.gz rvvm.tar.gz ${archive_hash}"
+auth_type='sha256'
+workdir="RVVM-${version}"
+depends=('sdl12-compat')
+
+build() {
+    run make OS=SerenityOS USE_SDL=1 USE_NET=1 GIT_COMMIT=76796ba all lib
+}
+
+install() {
+    run make OS=SerenityOS USE_SDL=1 USE_NET=1 GIT_COMMIT=76796ba DESTDIR="${DESTDIR}" install
```

Comment:
Something I missed in my last review: all these makeopts can go into `makeopts`, e.g.:
```bash
makeopts+=(
    'OS=SerenityOS',
    'USE_SDL=1',
    'USE_NET=1',
    'GIT_COMMIT=76796ba'
)
```

And then you can leave out your `install()` override, and your `build()` will become something like:
```bash
build() {
    run make "${makeopts[@]}" all lib
}
```

Reply:
Seems it doesn't. The Makefile is not prepared to understand such configuration where you magically swap target OSes and thinks you are crosscompiling for multiple targets.

![image](https://user-images.githubusercontent.com/50500857/234214147-72021ab1-00ab-4111-a98f-a99994c94506.png)

Reply:
Ah you are right, it uses `installopts` which you can set to the same value as `makeopts`.
https://github.com/SerenityOS/serenity/blob/868a3912f470c57181ef19ce0f83351f1e2ae5b3/Ports/.port_include.sh#L478

Reply:
> Also, the makeopts apparently should not have comas in it

Those commas were a syntactical mistake by me, should indeed be left out :-)

Reply:
Does this look fine to you?

```
makeopts+=(OS=SerenityOS USE_SDL=1 USE_NET=1 GIT_COMMIT=76796ba all lib)
installopts+=(${makeopts[@]})
*no build/install stages at all*
```

Reply:
Yes, apart from the fact that I'd put those elements inside of quotes!

```
build_opts=(
    'GIT_COMMIT=76796ba'
    'OS=SerenityOS'
    'USE_NET=1'
    'USE_SDL=1'
)
makeopts+=("${build_opts[@]}" 'all' 'lib')
installopts+=("${build_opts[@]}")
```

Something like that maybe?

Reply:
CI is failing now :C

Reply:
> "the authorized way"

Feel free to document some of this! We are always in need of more concise contribution guidelines such as these :^)

With your latest changes, it seems that it builds and installs correctly!